apps/contact-service/src/routes/clubs/member/getClubContacts.ts

```diff
@@ -0,0 +1,57 @@
+import {NotFoundError} from '@vexl-next/domain/src/general/commonErrors'
+import {
+  GetClubContactsErrors,
+  UserNotClubMemberError,
+} from '@vexl-next/rest-api/src/services/contact/contracts'
+import {GetClubContactsEndpoint} from '@vexl-next/rest-api/src/services/contact/specification'
+import makeEndpointEffect from '@vexl-next/server-utils/src/makeEndpointEffect'
+import {validateChallengeInBody} from '@vexl-next/server-utils/src/services/challenge/utils/validateChallengeInBody'
+import {Effect} from 'effect'
+import {Handler} from 'effect-http'
+import {ClubMembersDbService} from '../../../db/ClubMemberDbService'
+import {ClubsDbService} from '../../../db/ClubsDbService'
+
+export const getClubContacts = Handler.make(GetClubContactsEndpoint, (req, _) =>
+  makeEndpointEffect(
+    Effect.gen(function* (_) {
+      yield* _(validateChallengeInBody(req.body))
+
+      const clubsDb = yield* _(ClubsDbService)
+      const clubMembersDb = yield* _(ClubMembersDbService)
+
+      const club = yield* _(
+        clubsDb.findClubByUuid({uuid: req.body.clubUuid}),
+        Effect.flatten,
+        Effect.catchTag('NoSuchElementException', (e) => new NotFoundError())
+      )
+
+      const memberData = yield* _(
+        clubMembersDb.findClubMemberByPublicKey({
+          publicKey: req.body.publicKey,
+        }),
+        Effect.flatten,
+        Effect.catchTag('NoSuchElementException', (e) => new NotFoundError())
```

Comment:
Also, I think it's better to ensure user belongs to a club like this. It's better to not give user an information about club if they are not a member...

```suggestion
        Effect.catchTag('NoSuchElementException', new NotFoundError({message: 'Member not found'})),
                Effect.filterOrFail(
          (club) => club.id === member.clubId,
          () => new NotFoundError({message: 'Club not found'})
        )
```